src/main/java/org/mozilla/jss/ssl/PrintOutputStreamWriter.java

```diff
@@ -13,28 +13,28 @@
 
 
 
-class PrintOutputStreamWriter 
-    extends java.io.OutputStreamWriter
+class PrintOutputStreamWriter
+	extends java.io.OutputStreamWriter
 {
 
-    public PrintOutputStreamWriter(OutputStream out)
-    {
-    	super(out);
-    }
+public PrintOutputStreamWriter(OutputStream out)
```

Comment:
I'd reasonably expect this to be indented, do others agree? I think this is a C++-ism more so than a Java-ism. Thoughts?

Reply:
Agree. I think we probably should stick with Eclipse for autoformatting Java code, and leave C/Python code unchanged for now.

Reply:
Did you see the issue with Eclipse in the Java code though? It completely murdered a nicely formatted enum declaration.

Reply:
Ah yes, that's really bad.